src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -2143,20 +2143,33 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
             textData += text;
         }
 
+        bool useGlobal = formats == nullptr;
+        std::optional<bool> overrideHTML = std::nullopt;

```

Comment:
whait what are these `override*` variables used for? It _looks_ like they _should_ be used below, but I can't make sense of that conditional

Reply:
Added a comment to make it clearer.

Copy action now has a `copyFormatting` arg that overrides the global setting. This is passed in as the `formats` arg for `TermControl::CopySelectionToClipboard()`. If it's nullptr, we just let TermApp decide with the global setting. Otherwise, override that value for this copy operation and only include the desired formats.

Reply:
Yea but like, they're just unused variables right? Unless I'm just not seeing where they're used

Reply:
oh. Now I remember. I used them to debug this and see if formats was working properly in VS. My bad! Removed.